### PR TITLE
fix: Disable Google Credentials Sign In leading to crash

### PR DESCRIPTION
**Pull request by addiegupta**
Fixes: #1531

Changes:

- Removed google sign in features due to buggy implementation.

Screenshots for the change:

### CODE REVIEW

File: app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt [hunk 1]
```diff
@@ -23,9 +23,9 @@ const val RC_CREDENTIALS_SAVE = 3
 
 class SmartAuthViewModel : ViewModel() {
     private var requestedCredentialsEarlier = false
-    private lateinit var credentialsClient: CredentialsClient
-    private lateinit var signInClient: GoogleSignInClient
-    private lateinit var thisActivity: Activity
+    private var credentialsClient: CredentialsClient? = null
+    private var signInClient: GoogleSignInClient? = null
+    private var thisActivity: Activity? = null
```
Comment (iamareebjamal) [inline app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt]: This is a very very poor implementation of ViewModel. I don't know what was I smoking when I let this get merged.

There are Android classes used in ViewModel. No doubt there was a crash :man_facepalming: 

Remove this feature completely and create an issue to reimplement this

Don't delete this ViewModel

Comment (addiegupta) [reply app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt]: removed

Comment (iamareebjamal) [reply app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt]: Revert this

Comment (addiegupta) [reply app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt]: Done

**Comment by addiegupta** [comment]
@iamareebjamal Is anything else required in this?

**Pull request merged**